src/rollover.py

```diff
@@ -246,10 +247,8 @@ def wait_for_all_services(ecs_client, services_on_instance, service_events):
         service_events[service_id].append(event)
 
         if not completed:
-            service = service_descriptions[service_id]
-            print "ERROR: Timeout while waiting for %s to reach steady state" % (service['serviceName'])
-            return False
```

Comment:
sorry, I didn't mention it in the comments or description, but yes, I figured that would be a nice side effect to only list the ones that failed (and therefore to wait for all of them).